icu4c/source/i18n/unicode/numsys.h

```diff
@@ -20,15 +20,12 @@
 
 #include "unicode/utypes.h"
 
-#ifndef U_HIDE_INTERNAL_API
```

Comment:
I think this one might also have been fixed in the 64.2 update.
You might need to checkout master, fetch/pull, then rebase your branch on the updated master and resolve conflicts. (oh git.)